### ISSUE TITLE
NR Officium Defunctorum - neume transcription error?

### ISSUE THREAD

**Issue by Suinesra** [Nocturnale-Romanum/nocturnale-romanum#53]
Sandhofe: Nocturnale Romanum
Officium Defunctorum
Antiphon: Sitivit anima mea , page [198]

I would propose that the melodic flow of the first phrase would sound better if the punctum over the word "ad" was sung on Do, rather than on Re.

**Comment by xmarteo** [comment]
Agreed - antiphon modified, plus another change on the Deum that follows, following the restitution of this A/ by Pothier. See e8627a661f2dcde200c2568cb5b09f114daa7285
